contentcuration/contentcuration/utils/assessment/markdown.py

```
@@ -0,0 +1,120 @@
+
+    if not silent:
+        token = state.push("math_block", "math", 0)
+        token.block = True
+        token.content = match.group(1)
+        token.markup = "$$"
+
+    # Advance to next line after the math block
+    endpos = begin + match.end() - 1
+    line = begLine
+    while line < endLine:
+        if endpos >= state.bMarks[line] and endpos <= state.eMarks[line]:
```

Pylint (3.3.7)

[refactor] Simplify chained comparison between the operands

(R1716)